src/tools/compiletest/src/common.rs

```diff
@@ -269,10 +269,10 @@ pub struct Config {
     pub runtool: Option<String>,
 
     /// Flags to pass to the compiler when building for the host
-    pub host_rustcflags: Option<String>,
+    pub host_rustcflags: Option<Vec<String>>,
 
     /// Flags to pass to the compiler when building for the target
-    pub target_rustcflags: Option<String>,
+    pub target_rustcflags: Option<Vec<String>>,
```

Comment:
Unless we use the option-ness somewhere, I think making these just `Vec<String>` and empty if none are provided makes sense to me.

Reply:
Good call -- that will help simplify much of the logic